TripPlanner/Models/Trip.m

```diff
@@ -0,0 +1,103 @@
+//
+//  Trip.m
+//  TripPlanner
+//
+//  Created by Adrienne Li on 7/13/20.
+//  Copyright © 2020 ahli. All rights reserved.
+//
+
+#import "Trip.h"
+
+@implementation Trip
+
+@dynamic tripID;
+@dynamic author;
+@dynamic guests;
+@dynamic images;
+@dynamic description;
+@dynamic title;
+@dynamic location;
+@dynamic startDate;
+@dynamic endDate;
+
++ (nonnull NSString *)parseClassName {
+    return @"Trip";
+}
+
++ (NSString *)dateToString: (NSDate *)createdAt {
```

Comment:
this looks like a common enough function that is generic, such that it doesn't have to be on the trip model class. Consider putting it into a separate class.